Test/src/main/resources/mods.groovy

```diff
@@ -11,11 +11,32 @@ ModsDotGroovy.make {
         authors = ['Matyrobbrt', 'Paint_Ninja']
         credits = "${buildProperties.someProperty}"
 
-        customProperty = 'hello'
+        onForge {
+            customProperty = 'hello'
+        }
 
         dependencies {
             minecraft = 1.19..1.20 // equivalent to `minecraft = '[1.19,1.20)'`
-            forge = '[43.0.0,)' // equivalent to `forge { versionRange = '[43.0.0,)' }`
+            forge = '>=43.0.0' // equivalent to `forge { versionRange = '[43.0.0,)' }`
```

Comment:
Please keep the forge dependency in the mvnversion style, so that both ways are exemplified.